layouts/section/job-posting.en.html

```diff
@@ -1,145 +1,159 @@
-            <input
-              type="email"
-              name="email"
-              class="form-control validate-required validate-email"
-              id="email"
-              aria-required="true"
-              aria-describedby="email-required"
-            />
-            <span
-              id="email-required"
-              class="error-message validate-required-message"
-              style="display: none;"
-              >This field is required</span
-            >
-          </div>
+        <h2 class="h2-font-weight">Our hiring process</h2>
+        <h1 class="cds-title"> At CDS, we welcome and invite difference.</h1>
+        <p>The Canadian Digital Service (CDS) is changing how the federal government designs and delivers digital
+          services. We’re here to improve people’s lives by changing how the government builds technology. We do this by
+          putting people’s needs and concerns front and centre. Every experience Canadians have with their government
+          should be safe, fast, easy, transparent, and accessible.</p>
+        <p>We proudly, passionately, and actively strive to make CDS more reflective of the society that we serve. We
+          will only be able to deliver better, more accessible, and more inclusive public services if we harness diverse
+          thoughts, experiences, and skills. We work hard to create an environment where different perspectives and
+          experiences are valued. We are committed to helping diverse talent thrive.</p>
+        <p>CDS welcomes all applicants, including Veterans and people of all races, ethnicities, religions, sexual
+          orientations, gender identities and expressions, national origins, disabilities, ages, body sizes and
+          including those with diverse households and family commitments. We are committed to providing an inclusive and
+          barrier-free work environment, starting with the hiring process. If you need to be accommodated during any
+          phase of the evaluation process, please use the Contact information below to request specialized
+          accommodation. All information received in relation to accommodation will be kept confidential.</p>
 
-          <!-- Resume -->
-          <div class="form-group">
-            <label for="resume">Resume</label>
-            <input class="resume"
-              type="file"
-              id="resume"
-              name="resume"
-              accept="application/msword, application/pdf, text/plain" required
-            />
-          </div>
+        <div class="cta__link black">
+          <a href="/join-our-team/">More about our hiring process<svg width="16" height="16" viewBox="0 0 16 16"
+              xmlns="http://www.w3.org/2000/svg">
+              <path d="M8 9H0V7H8V0L16 8L8 16V9Z" />
+            </svg></a>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div id="apply-now" class="work-with-us--apply-now">
+  {{ if .Params.leverId }}
```

Comment:
What does it mean if there isn't a `leverId`? We'll link the `apply now` above down to an empty section?

Reply:
If there is no id then the job posting would not exist, right? On second thought, do we even need the conditional in that case?

Reply:
i've changed it so if leverID doesn't exist, the apply now button won't either (which makes sense to me?)